Dockerfile

```diff
@@ -25,21 +22,23 @@ RUN DEBIAN_FRONTEND=noninteractive && \
   libopenoffice-oodoc-perl \
   postgresql-client \
   ssmtp \
-  lsb-release
+  lsb-release \
+  && rm -rf /var/lib/apt/lists/*
 
 
 # Build time variables
 ENV LSMB_VERSION master
 ENV NODE_PATH /usr/local/lib/node_modules
-ENV DEBIAN_FRONTEND=noninteractive
 
 ARG CACHEBUST
 
 
 # Java & Nodejs for doing Dojo build
 # Uglify needs to be installed right before 'make dojo'?!
-RUN apt-get -y install git make gcc libperl-dev npm curl && \
+RUN apt-get update && \
```

Comment:
What about moving all of this run command other than the clone to the previous run line, and eliminate one apt-get update? If we're putting a cache-bust here, we might end up needing a dist-upgrade...